demo/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Web Components demo</title>
+
+  <!-- ultimately should be something like https://unpkg.com/attractions@2.2.0/dist/index.js -->
+  <script src="../attractions/dist/bundle.js"></script>
+</head>
+
+<body>
+  <a-card>
+    <a-button filled="true" danger="true">
+      Alert
+    </a-button>
+    <!-- camelCase props seem to not work (HTML is case-insensitive) -->
+    <a-switch slotLeft="true">
```

Comment:
maybe we should reconsider the way we name our boolean and string props to be compatible with this

Reply:
Or not rename, but add an "alias", perhaps

Reply:
is that even possible?

Reply:
I mean another prop that would act as a fallback. If `slotLeft` is `false` (or maybe `null` or some other default signifying its absence), check the value of `slotleft`. But that's kinda hacky and I don't like it.
Do we even have that many important props with camelCase names? I mean, people can still use `.$$set`